.ci/update-github-page.sh

```diff
@@ -72,24 +72,24 @@ CONTENT=$(cat github_post.txt)
 echo "$CONTENT"
 
 echo 'Updating content to be be json value friendly'
-UPDATED_CONTENT=$(awk '{printf "%s\\n", $0}' github_post.txt |  sed "s/\`/'/g")
+UPDATED_CONTENT=$(awk '{printf "%s\\n", $0}' github_post.txt | sed "s/\`/'/g" | jq -Rsa .)
```

Comment:
I think we can remove 
```
sed "s/\`/'/g"
```
At all, as it will not be considered in specific to bash

Reply:
Can you test it without it ?

Reply:
Doesn't make that big of a difference. Either works.

With:
![Screenshot from 2022-11-23 19-32-35](https://user-images.githubusercontent.com/23459549/203622715-29689dfd-1760-42de-9c83-cce5edc93529.png)

Without:
![Screenshot from 2022-11-23 19-34-21](https://user-images.githubusercontent.com/23459549/203622771-8cc01135-2047-4065-bdc3-73f8f81f544e.png)

I will remove it.

Reply:
Done.